### DISCUSSION TITLE
How setup Airflow Web login as Github OAUTH with multiple Webserver with load balancer

### DISCUSSION THREAD

**Discussion by uplsh580** [apache/airflow#25273] [Q&A]
hello.
I'm trying to configure the Github Oauth login setting in Airflow 2.2.5 version with the below conditions.
- 2 airflow webservers
- load balancer to access webs
And proceeded by referring to the guide below.
https://airflow.apache.org/docs/apache-airflow/2.2.5/security/webserver.html#example-using-team-based-authorization-with-github-oauth

When I follow this, I cannot login when I apply the homepage URL and call back URL of Github Oauth Application Setting page as the LB address.
<img width="920" alt="image" src="https://user-images.githubusercontent.com/50520567/180734491-409249d2-b84e-4a96-bd36-8e24c59aa54b.png">

There was no problem in setting up 1 webserver without LB.
Are there any other considerations when applying Github Oauth setting with LB and multiple web servers?
(There was no problem when using LB in db login.)

**Comment by potiuk** [comment]
I think you might look at Flask Application Builder documentation https://flask-appbuilder.readthedocs.io/en/latest/security.html and maybe ask the question there. We are using Flask App builder for authentication and there were some changes in the way how Oauth works and you might want to consider changes there and query how to solve your problems there.
